db/migrate/20230308160615_create_data_resources.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class CreateDataResources < ActiveRecord::Migration[7.0]
+  def change
+    create_table :data_resources, id: :uuid do |t|
+      t.text :identifier
+      t.text :title
+      t.text :description
+      t.text :keywords, array: true, default: []
+      t.text :theme
```

Comment:
theme should also be an array (they're similar to keywords except themes are IRIs not plain strings) - so perhaps it should be `themes`